control/vehicle_cmd_gate/src/vehicle_cmd_gate.cpp

```diff
@@ -496,9 +500,11 @@ AckermannControlCommand VehicleCmdGate::filterControlCommand(const AckermannCont
   AckermannControlCommand out = in;
   const double dt = getDt();
   const auto mode = current_operation_mode_;
+  const auto current_status_cmd = getActualStatusAsCommand();
+  const auto ego_is_stopped = std::abs(current_status_cmd.longitudinal.speed) < 1e-3;
 
   // Apply transition_filter when transiting from MANUAL to AUTO.
-  if (mode.is_in_transition) {
+  if (!ego_is_stopped && mode.is_in_transition) {
```

Comment:
@maxime-clem Sorry if I misunderstand, but I guess just adding the condition:

```
if (ego_is_stopped) {
    prev_values.longitudinal = current_status_cmd.longitudinal;
}
```

is enough. Why the `filter_on_transition_` is required when the ego is stopped?